fastlane/Fastfile

```diff
@@ -16,6 +16,8 @@ app_secret = ENV.fetch('STREAM_VIDEO_SECRET', nil)
 derived_data_path = 'derived_data'
 source_packages_path = 'spm_cache'
 buildcache_xcargs = 'CC=clang CPLUSPLUS=clang++ LD=clang LDPLUSPLUS=clang++'
+gci = 'com.googleusercontent.apps.347024607410-48j4atipav0tcr4pesap4elr1u9t11uh'
```

Comment:
As this is also in public doesn't it defy all the actions we took to hide it?. Can we use the environment variable?

Note: This is the reversed id. The regular one begins with the numbers